core/server/api/themes.js

```diff
@@ -26,11 +27,24 @@ var debug = require('debug')('ghost:api:themes'),
 themes = {
     browse: function browse() {
         debug('browsing');
-        var result = packageUtils.filterPackages(themeList.getAll());
+        var result = themeList.toAPI(themeList.getAll(), settingsCache.get('activeTheme'));
```

Comment:
Hopefully sharing my notes makes it possible to see it's all planned to change 😺